### PR TITLE
KOGITO-5784 Setup Quarkus platform internal nightly

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

**Comment by radtriste** [comment]
This would deploy the platform kogito bom: http://nexus3-kogito-tools.kogito-cluster-0ad6762cc85bcef5745bb684498c2436-0000.us-south.containers.appdomain.cloud/repository/maven-snapshots/io/quarkus/platform/quarkus-kogito-bom/kogito-2.0.0-SNAPSHOT/quarkus-kogito-bom-kogito-2.0.0-20230220.113943-1.pom

File: .ci/jenkins/Jenkinsfile.buildchain [hunk 1]
```diff
@@ -67,7 +67,9 @@ pipeline {
                         assert mavenDeployRepositoryCredsId : 'Please provide the maven deploy repository credentials'
                     }
 
-                    if (env.GIT_BRANCH_NAME) {
+                    if (params.DISPLAY_NAME) {
+                        currentBuild.displayName = params.DISPLAY_NAME
+                    } else if (env.GIT_BRANCH_NAME) {
                         currentBuild.displayName = env.GIT_BRANCH_NAME
                     }
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.buildchain]:
```suggestion
                    currentBuild.displayName = params.DISPLAY_NAME ?: env.GIT_BRANCH_NAME ?: currentBuild.displayName
```

**Review by Ginxo — APPROVED** [review]
LGTM, just a technical suggestion

File: .ci/jenkins/Jenkinsfile.buildchain (hunk 1 above)
Comment (radtriste) [reply .ci/jenkins/Jenkinsfile.buildchain]: do you it more readable ? :D

Comment (radtriste) [reply .ci/jenkins/Jenkinsfile.buildchain]: anyway, that is a oneliner. will take it

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**